_std/defines/fluid_pipes.dm

```diff
@@ -0,0 +1,4 @@
+#define FLUIDPIPE_NORMAL 1
```

Comment:
at least move to a `fluid.dm` or something

Reply:
done!